department-of-veterans-affairs/caseflow
# Hearing worksheet downloader link

connect #3601
<img width="1287" alt="screen shot 2017-12-14 at 4 19 41 pm" src="https://user-images.githubusercontent.com/1745984/34014862-a309a030-e0ea-11e7-83fa-14fc2c7dda2b.png">

To test:
- Load welcome gate, click on hearing worksheet, confirm it loads correctly
- Confirm reader users cannot access other hearing prep pages
- Confirm hearing prep users can still access print page
